hackerrank/1-week-prep/day-3/zig-zag-sequence.py

```diff
@@ -0,0 +1,18 @@
+def find_zig_zag_sequence(a, n):
+    a.sort()
+    mid = int((n + 1)/2)-1
+    a[mid], a[n-1] = a[n-1], a[mid]
+
+    st = mid + 1
+    ed = n - 2
```

Comment:
I do not condone this way of naming variables.

Reply:
https://help.hackerrank.com/hc/en-us